File: codewars/Js/BitCounting.js
let countBits = function(n) {

    let count = 0;
    let binaryString = n.toString(2); //이진수로 변환

    for (let i = 0; i <binaryString.length ; i++) {

        if( binaryString[i] === '1') {
            count++;
        }
    }
    return count;
};

// 솔루션2

File: codewars/Java/BitCounting.java
public class BitCounting {
    public static int countBits(int n) {
        int count = 0;
        String value = Integer.toBinaryString(n);

        for (int i = 0; i <value.length(); i++) {
            char value1 = value.charAt(i);
            if(value1 == '1') {
                count++;
            }
        }
        return count;
    }
}
